### PR TITLE
Update sharing-workflows-with-your-organization.md

### PR DESCRIPTION
**Pull request by MaxFlanders**
### Why:

Documentation didn't highlight an important piece of information effectively.

### What's being changed:

Placing a small piece of documentation in a callout box so it stands out more.

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [x] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by Syahril92 — APPROVED** [review]

**Review by Syahril92 — CHANGES_REQUESTED** [review]
Good n nice

**Comment by janiceilene** [comment]
@MaxFlanders Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/actions/learn-github-actions/sharing-workflows-with-your-organization.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ If you need to share workflows and other {% data variables.product.prodname_acti
 
 ### Creating a workflow template
 
-Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows. Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
+Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows.
+
+> **Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
```
Comment (ajs256) [inline content/actions/learn-github-actions/sharing-workflows-with-your-organization.md:26]:
```suggestion
{% note %}
**Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
{% endnote %}
```

**Review by ajsirota — COMMENTED** [review]
I just added a suggestion to use a callout instead of a blockquote.

**Review by anh1200a — APPROVED** [review]

File: content/actions/learn-github-actions/sharing-workflows-with-your-organization.md (hunk 1 above)
Comment (anh1200a) [reply content/actions/learn-github-actions/sharing-workflows-with-your-organization.md]: Ok

**Review by anh1200a — DISMISSED** [review]

**Review by anh1200a — COMMENTED** [review]
Okr

File: content/actions/learn-github-actions/sharing-workflows-with-your-organization.md [hunk 2]
```diff
@@ -21,7 +21,11 @@ If you need to share workflows and other {% data variables.product.prodname_acti
 
 ### Creating a workflow template
 
-Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows. Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
+Workflow templates can be created by users with write access to the organization's `.github` repository. The templates can then be used by organization members who have permission to create workflows.
+
+{% note %}
+**Note:** Workflow templates can be used to create new workflows in an organizations' public repositories; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
```
Comment (mchammer01) [inline content/actions/learn-github-actions/sharing-workflows-with-your-organization.md:27]:
```suggestion

**Note:** Workflow templates can be used to create new workflows in an organization' s public repository; to use templates to create workflows in private repositories, the organization must be part of an enterprise plan.
```

File: content/actions/learn-github-actions/sharing-workflows-with-your-organization.md [hunk 3]
```diff
@@ -21,7 +21,11 @@ If you need to share workflows and other {% data variables.product.prodname_acti
...
+{% endnote %}
```
Comment (mchammer01) [inline content/actions/learn-github-actions/sharing-workflows-with-your-organization.md:28]:
```suggestion

{% endnote %}
```

**Review by mchammer01 — APPROVED** [review]
@MaxFlanders - LGTM ✨ 
(Made a couple of minor syntax and editorial edits)

**Pull request merged**